examples/applications/author-disambiguation/utils.py

```diff
@@ -0,0 +1,282 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Beard.
+# Copyright (C) 2015 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Helpers for ML applications."""
+
+import json
+
+
+from beard.utils import name_initials
+from beard.utils import normalize_name
+
+
+def load_signatures(signatures_filename, records_filename):
+    """Load signatures from json fiels.
+
+    Parameters
+    ----------
+    :param signatures_filename: string
+        Path to the signatures file
+
+    :param records_filename: string
+        Path to the records file.
+
```

Comment:
We have to say here that the file must be in a json format, same for signatures.. similar to what mentioned in lines 193, 194